training/datastore/utils.py

```diff
@@ -168,17 +168,19 @@ def extract_package_manifest_lst(self, data):
         return all_packages, list_of_manifest_list
 
     def create_content_matrix(self, pkg_tag_map,
-                              all_packages, vocabulary, tag_idx_map):  # pragma: no cover
+                              all_packages, vocabulary):  # pragma: no cover
         """Create Content Matrix."""
-        pkg_tag_map = self.dict_
-        all_packages = self.set_
-        vocabulary = self.set_
-        tag_idx_map = self.dict_
+        tag_idx_map = self.create_vocabulary_map(vocabulary)
         content_matrix = np.zeros([len(all_packages), len(vocabulary)])
         if tag_idx_map:
             for idx, package in enumerate(all_packages):
-                package_tags = [tag_idx_map[tag] for tag in pkg_tag_map[package]]
-                content_matrix[idx, package_tags] = 1
+                try:
+                    package_tags = [tag_idx_map[tag] for tag in pkg_tag_map[package]]
+                    if idx == 0:
+                        logger.info("Setting to 1: {}".format(package_tags))
+                    content_matrix[idx, package_tags] = 1
+                except Exception:
```

Comment:
If you know which `Exception` will be thrown here it is best to catch that specific exception instead of a generic `Exception` as otherwise unexplained behavior goes unnoticed.

Reply:
Ok, will figure it out.